docs/release-notes/2.25.0.md

```diff
@@ -0,0 +1,78 @@
+# Percona Monitoring and Management 2.25.0
+
+<table class="docutils field-list" frame="void" rules="none">
+  <colgroup>
+    <col class="field-name">
+    <col class="field-body">
+  </colgroup>
+  <tbody valign="top">
+    <tr class="field-odd field">
+      <th class="field-name">Date:</th>
+      <td class="field-body">December 7, 2021</td>
+    </tr>
+    <tr class="field-even field">
+      <th class="field-name">Installation:</th>
+      <td class="field-body">
+        <a class="reference external" href="https://www.percona.com/software/pmm/quickstart">Installing Percona Monitoring and Management</a></td>
+    </tr>
+  </tbody>
+</table>
+
+Percona Monitoring and Management (PMM) is a free and open-source platform for managing and monitoring MySQL, MongoDB, and PostgreSQL performance.
+
+## Release Highlights 
+- **Percona Platform (Technology Preview):** 
+    - Ability to connect your Percona Monitoring and Management instances to your Percona Platform to get the latest intelligence from Percona and have all your accounts and relations with Percona centralized. READ_MORE
+
+- **Enhanced PostgreSQL monitoring**
+    - From this release, PMM will let users who add PostgreSQL Servers be monitored by PMM to [specify database name for monitoring](https://deploy-preview-643--pmm-doc.netlify.app/details/commands/pmm-admin.html#postgresql). Previous versions have the default `postgres` database, which wasn't always convenient for the users. 
+    - This Release of PMM is supported and compatible with the recent release of the PostgreSQL extension - `pg_stat_monitor` v1.0.0. You can read more about its features and advantages in this blog post (LINK to pg_stat_monitor release announcement or Userguide)
+    - PMM 2.25 also be compatible and working fine with the latest Percona Disctibutions for PostgreSQL released today (LINK), which includes the newest version of pg_stat_monitor extension.  
+
+- **Grafana usability enhancements**
+    - We changed the way how the dashboards, plugins, and data sources are added into PMM. We now use the native Grafana provisioning mechanism in PMM. This change will lead to a faster and more reliable update process for our users from one version of PMM to another. With this update, we'll preserve all custom changes you might already do with plugins and dashboards, but we'll encourage you to be back up them before an upgrade and recheck after, just in case. 

```

Comment:
```suggestion
    - We changed the way that dashboards, plugins, and data sources are added into PMM. We are now using the native Grafana provisioning mechanism in PMM. This change will lead to a faster and more reliable update process for our users moving from one version of PMM to another. With this update, we'll preserve all custom changes you have already done with plugins and dashboards, but we still encourage you to back them up before an upgrade and recheck after, just in case. 

```